bin/foreman-maintain

```diff
@@ -4,6 +4,11 @@ $LOAD_PATH.unshift(File.expand_path('../../lib', __FILE__))
 
 require 'foreman_maintain'
 
+CONFIG_FILE = '/etc/foreman-maintain/config/foreman_maintain.yml'.freeze
+STORAGE_DATA_FILE = '/lib/foreman-maintain/data.yml'.freeze
```

Comment:
@iNecas,
Yes, we have config files for git. 
Did you mean we need to declare separate config files for packaging like test environment, right?
For example, foreman_maintain.yml.packaging
Please correct me.